src/sentry/api/endpoints/project_rule_actions.py

```diff
@@ -0,0 +1,63 @@
+from rest_framework.exceptions import ValidationError
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import ProjectAlertRulePermission, ProjectEndpoint
+from sentry.api.serializers.rest_framework import RuleActionSerializer
+from sentry.models import Rule
+from sentry.rules.processor import RuleProcessor
+from sentry.utils.safe import safe_execute
+from sentry.utils.samples import create_sample_event
+from sentry.web.decorators import transaction_start
+
+
+@region_silo_endpoint
+class ProjectRuleActionsEndpoint(ProjectEndpoint):
+    private = True
+    permission_classes = (ProjectAlertRulePermission,)
+
+    @transaction_start("ProjectRuleActionsEndpoint")
+    def post(self, request: Request, project) -> Response:
+        """
+        Creates a dummy event/group and activates the actions given by request body
+
+            {method} {path}
+            {{
+                "actions": []
+            }}
+
+        """
+        serializer = RuleActionSerializer(
+            context={"project": project, "organization": project.organization}, data=request.data
+        )
+
+        if not serializer.is_valid():
+            raise ValidationError
```

Comment:
We should be able to extract the failed validations from the serializer and return those.